ExomeDepth/make_BEDdetail.py

```diff
@@ -208,7 +217,7 @@ def slice_vcf(args, merge_dic):
                 event_dic[event][sampletype]["deldupratio"].append(deldupratio)
                 event_dic[event][sampletype]["totalcalls"].append(totalcalls)
                 event_dic[event][sampletype]["gender"].append(gender)  # Not used in output at the moment.
-
+                event_dic[event][sampletype]["refset"].append(refset)
     all_event_file.close()
```

Comment:
here I see a file.close() with an `= open()` earlier in your file. However it is best practise (python 3) to use `with open()`, like you also did in line 60 for `vcf_input_file`. python will automatically close the file at the right time.
might be an improvement to make this more uniform and more python3.

Reply:
Good point, changed.